### ISSUE TITLE
Make a chromium version

### ISSUE THREAD

**Issue by wpsumo** [tiborbarsi/image-video-block-browser-addon#29]
Please consider making a chromium version as well.

**Comment by ghost** [comment]
YES PLEASE!!! Chromium literally has no extensions comparable to this.

**Comment by nazsol** [comment]
I think i made the same request on the Firefox extensions page, but yeah, there isn't anything else out there like this and would love a Chrome version.
